src/Network/Email/AbstractTransport.php

```diff
@@ -1,3 +1,11 @@
 <?php
-// @deprecated Backward compatibility with 2.x, 3.0.x
-class_alias('Cake\Mailer\AbstractTransport', 'Cake\Network\Email\AbstractTransport');
+namespace Cake\Network\Email;
+
+if(false) {
+    /**
+     * @deprecated Use Cake\Mailer\AbstractTransport instead.
+     */
+    abstract class AbstractTransport extends \Cake\Mailer\AbstractTransport
```

Comment:
@markstory All the classes have the same name. If I add a "use" then I have to give it an alias and it just seems confusing after that.